feature/gribi/otg_tests/ack_in_presence_other_routes/route_ack_test.go

```diff
@@ -227,21 +227,14 @@ func configureNetworkInstance(t *testing.T) {
 }
 
 // configStaticRoute configures a static route.
-func configStaticRoute(t *testing.T, dut *ondatra.DUTDevice, prefix string, nexthop string) *telemetry.NetworkInstance_Protocol_Static {
-	d := &telemetry.Device{}
-	ni1 := d.GetOrCreateNetworkInstance(*deviations.DefaultNetworkInstance)
-	ni1.Enabled = ygot.Bool(true)
-	ni1.Name = ygot.String(*deviations.DefaultNetworkInstance)
-	ni1.Description = ygot.String("Static route added by gNMI-OC")
-	static := ni1.GetOrCreateProtocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "STATIC")
+func configStaticRoute(t *testing.T, dut *ondatra.DUTDevice, prefix string, nexthop string) {
```

Comment:
Please do update the corresponding the ATE file with the changes here as well.

Reply:
I have modified the ATE test as well. Please see if this works as expected.

Reply:
Please wait for #600. I believe that will address the concern here.

Reply:
Keep the *deviations.StaticProtocolName change that was picked up in the rebase.

Reply:
Using *deviations.StaticProtocolName now